src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php

```diff
@@ -121,30 +113,12 @@ public function finishView(FormView $view, FormInterface $form, array $options)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        // BC clause for the "intention" option
-        $csrfTokenId = function (Options $options) {
-            return $options['intention'];
```

Comment:
we were missing a deprecation notice when the user passes ``intention`` rather than ``csrf_token_id`` though